src/routers/companyRouter.ts

```diff
@@ -46,15 +47,19 @@ router.get(config.CONFIRM_URL, async (req: Request, res: Response, next: NextFun
 });
 
 router.post(config.CONFIRM_URL, async (req: Request, res: Response, next: NextFunction) => {
-  const companyProfile: CompanyProfile | undefined = req.session?.getExtraData("companyProfile");
+  const companyProfile: CompanyProfile | undefined = req.session?.getExtraData(constants.COMPANY_PROFILE);
```

Comment:
```suggestion
  const companyProfile: CompanyProfile | undefined = req.session?.getExtraData(COMPANY_PROFILE);
```

Reply:
done - replaces all const.* statements with the actual constants